### PR TITLE
Document the release process in the cliquet documentation

### PR DESCRIPTION
**Pull request by ametaireau**
@Natim r?

### CODE REVIEW

File: cliquet_docs/contributing.rst [hunk 1]
```diff
@@ -112,3 +112,35 @@ on ``#storage`` on ``irc.mozilla.org``.
...
   channel on the mozilla IRC network.
+
+
+How to release
+==============
+
+In order to prepare a new release, we are following the following steps:
+
+1. Update the CHANGELOG with the changes since the last release;
+2. Update the version in setup.py;
```
Comment (Natim) [inline cliquet_docs/contributing.rst]: We are using pre-release from zest.releaser for that

Comment (Natim) [inline cliquet_docs/contributing.rst]: Here are the steps we are following for Kinto. https://github.com/Kinto/kinto/pull/137

Comment (ametaireau) [inline cliquet_docs/contributing.rst]: I personally don't because it doesn't fit well with our release process in my opinion. I will add a note about it.

Comment (Natim) [inline cliquet_docs/contributing.rst]: I like it to be clear and it prevent us for doing mistakes (forgetting the Back to development commit for instance)

**Pull request merged**